app/models/item.rb

```diff
@@ -220,7 +220,7 @@ def audited_attributes
   end
 
   # called when item is updated
-  def audited_changes
+  def audited_changes(**args)
```

Comment:
This started taking args in https://github.com/collectiveidea/audited/pull/657 so every Item interaction started throwing `ArgumentError`s. Since we call `super` with no args I believe they should automatically be passed along, so it's fine to just capture the args to make the method signature match the callers.